OneCommunityGlobal/HighestGoodNetworkApp
# Xiao F Vertically align partial label of team code filter dropdown

# Description
![CleanShot 2023-10-13 at 15 54 37](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/43768723/2ce69a01-370c-42ca-ad24-19181498eeeb)


## Related PRS (if any):
N/A

## Main changes explained:
- Update file `src/components/WeeklySummariesReport/WeeklySummariesReport.css` to use monospaced font to align numbers in parenthesis vertically

## How to test:
1. Check into current branch
2. Do `npm install` and `npm run start:local` to run this PR locally
3. Log in as admin/owner user
4. Go to Reports→ Weekly Summaries Report→ Select Team Code
5. Verify that numbers in parenthesis are vertically aligned equally for all 4-letter codes and 5-letter codes

## Screenshots or videos of changes:
Before:
![CleanShot 2023-10-13 at 16 03 40](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/43768723/45aa73a7-03c1-42d7-a4a1-6b8e83fe5fcc)

After:
![CleanShot 2023-10-13 at 16 01 55](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/43768723/18284243-be5c-4d92-b3bd-216b9cb9e7e7)

## Note:
I tried to put HTML tags and stylings into teamCodes's label variable, but I found that it conflicts with what is defined in `react-multi-select-component` package. So this is a workaround to make alignment work.
